content/docs/introduction.md

```diff
@@ -16,10 +16,16 @@ toc: false
  * Download [graphwalker-cli](http://graphwalker.github.io/download/)
 
 ## Test idea and design
-The purpose of the test design is to describe the **expected behavior of the system under test**. The way it works, is that you in a finite state diagram [model], express an action as a directed edge. An edge is also known as an arrow, arc or transition. The edge points to a vertex. Also known as a node or state, where the results or the consequence of the previous action is verified/asserted.
+The purpose of the test design is to describe the **expected behavior of the system under test**
```

Comment:
I have transferred some sentences on new lines due to some lines are very long and it is not readable in IDE.
It requires horizontal scrolling and anybody who read it may lost focus on which line he was.